modules/kafka/kafka.go

```diff
@@ -125,6 +125,7 @@ func Run(ctx context.Context, img string, opts ...testcontainers.ContainerCustom
 func copyStarterScript(ctx context.Context, c testcontainers.Container) error {
 	if err := wait.ForListeningPort(publicPort).
 		SkipInternalCheck().
+		SkipExternalCheck().
```

Comment:
suggestion: looking at this I think a helper would be nice e.g. `ForMappedPort(<port>)` which sets skip internal and external.

For example add to `host_port.go`
```go
func ForMappedPort(port nat.Port) *HostPortStrategy {
	return NewHostPortStrategy(port).SkipInternalCheck().SkipExternalCheck()
}
```

Reply:
Done